src/core/api.ts

```diff
@@ -91,14 +91,23 @@ export class Api<R extends RawApi = RawApi> {
         readonly installedTransformers: () => Transformer<R>[];
     };
 
+    /**
+     * Constructs a new instance of `Api`. It is independent from all other
+     * instances of this class. For example, this lets you install a custom set
+     * if transformers.
+     *
+     * @param token Bot API token obtained from [@BotFather](https://t.me/BotFather)
+     * @param options Optional API client options for the underlying client instance
+     * @param webhookReplyEnvelope Optional envelope to handle webhook replies
+     */
     constructor(
-        token: string,
-        config?: ApiClientOptions,
+        public readonly token: string,
+        public readonly options?: ApiClientOptions,
```

Comment:
Wer should probably do
```suggestion
        public readonly options?: Readonly<ApiClientOptions>,
```

Reply:
I'm naïve enough to believe that this isn't needed